lib/chusaku.rb

```diff
@@ -52,15 +54,25 @@ def self.call
       end
 
       # Write to file.
-      parsed_content = parsed_file.map { |pf| pf[:body] }
-      write(path, parsed_content.join)
+      parsed_content = parsed_file[:groups].map { |pf| pf[:body] }
+      new_content = parsed_content.join
+      if parsed_file[:content] != new_content
+        write(path, new_content) unless args.include?('--dry-run')
+        annotated_paths << path
+      end
     end
 
     # Output results to user.
-    if controller_paths.any?
-      puts "Annotated #{controller_paths.join(', ')}"
+    if annotated_paths.any?
+      puts "Annotated #{annotated_paths.join(', ')}"
+      if args.include?('--exit-with-error-on-annotation')
```

Comment:
Is the intended use of this flag with CI so that CI can fail if there are annotations that still need to be made? Based on your PR description, that's what I'm gathering, but just want to double check.